tests/util/bft.py

```diff
@@ -44,9 +44,9 @@ def interesting_configs(
         selected=lambda *config: True):
 
     bft_configs = [{'n': 4, 'f': 1, 'c': 0, 'num_clients': 4},
-                   {'n': 7, 'f': 2, 'c': 0, 'num_clients': 4},
                    {'n': 6, 'f': 1, 'c': 1, 'num_clients': 4},
-                   # {'n': 9, 'f': 2, 'c': 1, 'num_clients': 4},
+                   {'n': 7, 'f': 2, 'c': 0, 'num_clients': 4},
+                   # {'n': 9, 'f': 2, 'c': 1, 'num_clients': 4}
```

Comment:
Note: I haven't changed anything significant here - just sorted the entries by ascending _n_